### ISSUE TITLE
Crash on  same --outputdir

### ISSUE THREAD

**Issue by carissableker** [tseemann/prokka#379]
I had a crash when running multiple prokka tasks at the same time with the same output folder but different prefixes. The issue was that each task created/used the file `Escherichia.blast` in the output folder, and I assume this caused clobbering.

My fix was to make separate output folders for each of my tasks and move the results over after. A real fix would be to have prokka make unique file names.

One command was: 
```
prokka --outdir /.../project_scratch/prokka_annotations/ --force --prefix GCA900096845 --addgenes --locustag GCA900096845 --genus Escherichia --species coli  --usegenus --cpus 1 /.../GCA_900096845.1_genomic.fna
```

A portion of the log file for a single task, up to the crash is as follows:
```
[09:52:36] This is prokka 1.13.3
[09:52:36] Written by Torsten Seemann <torsten.seemann@gmail.com>
[09:52:36] Homepage is https://github.com/tseemann/prokka
[09:52:36] Local time is Mon Apr 15 09:52:36 2019
[09:52:36] You are ...
[09:52:36] Operating system is linux
[09:52:36] You have BioPerl 1.007002
[09:52:36] System has 64 cores.
[09:52:36] Will use maximum of 1 cores.
[09:52:36] Annotating as >>> Bacteria <<<
[09:52:36] Re-using existing --outdir /.../prokka_annotations/A/
[09:52:36] Using filename prefix: GCA002125925.XXX
[snip]
09:53:25] Will use blast to search against /.../blast/db/genus/Escherichia with 1 CPUs
[09:53:25] Running: cat \.../project_scratch\/prokka_annotations\/A\/\/Escherichia\.faa | parallel --gnu --plain -j 1 --block 724153 --recstart '>' --pipe blastp -query - -db /.../blast/db/genus/Escherichia -evalue 1e-09 -qcov_hsp_perc 80 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \...\/prokka_annotations\/A\/\/Escherichia\.blast 2> /dev/null
```

The error that is thrown is:
```
------------- EXCEPTION: Bio::Root::Exception -------------
MSG: no data for midline Reference for composition-based statistics: Alejandro A. Schaffer,
STACK: Error::throw
STACK: Bio::Root::Root::throw /.../envs/blast/lib/site_perl/5.26.2/Bio/Root/Root.pm:447
STACK: Bio::SearchIO::blast::next_result /.../envs/blast/lib/site_perl/5.26.2/Bio/SearchIO/blast.pm:1932
STACK: /.../envs/blast/bin/prokka:1068
-----------------------------------------------------------
```

Although the line starting with `MSG: no data for midline` randomly changes.

**Comment by tseemann** [comment]
@carissableker instead of `--usegenus` can you try `--proteins ../project_scratch\/prokka_annotations/A/Escherichia.faa`  ?

**Comment by carissableker** [comment]
`Escherichia.faa`  was created/put there by prokka, so I'm not sure if that would help in general?

**Comment by tseemann** [comment]
Sorry for the slow reply @carissableker 

"multiple prokka tasks at the same time with the same output folder but different prefixes".

I had never anticipated that.  Here is the existing code I wrote long, long ago:

```
    # create a unqiue output name so we can save them in --debug mode
    my $outname = $db->{DB};
    $outname =~ s{^.*/}{};
    # we write out all the CDS which haven't been annotated yet and then search them
    my $faa_name = "$outdir/$outname.faa";
```

It's not great. I will make the names truly unique (used PID maybe) or use File::Temp.

**Comment by tseemann** [comment]
I have fixed in 1.14.1 - thanks for the bug report, and my apologies for not understanding your clearly described issue at the start.
